tools/rules/pex_rules.bzl

```diff
@@ -169,12 +170,15 @@ def pex_test_impl(ctx):
   pex_test_files = pex_test_file_types.filter(pex_file_types.filter(transitive_sources))
   test_run_args = ' '.join([f.path for f in pex_test_files])
 
+  arguments = ["--disable-cache"]
```

Comment:
could we instead set this in common_pex_arguments() so we don't have to set it in two places?